services/thumbnails/pkg/service/http/v0/service.go

```diff
@@ -73,11 +73,12 @@ func (s Thumbnails) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // GetThumbnail implements the Service interface.
 func (s Thumbnails) GetThumbnail(w http.ResponseWriter, r *http.Request) {
+	logger := s.logger.SubloggerWithRequestID(r.Context())
 	key := r.Context().Value(keyContextKey).(string)
 
 	thumbnail, err := s.manager.GetThumbnail(key)
 	if err != nil {
-		s.logger.Error().
+		logger.Error().
```

Comment:
@C0rby @butonic I think this could be level `warn`. Could the admin do anything about it when we land in this codepath?

Reply:
```suggestion
		logger.Debug().
```